app/models/mixins/dialog_option_mixin.rb

```diff
@@ -0,0 +1,33 @@
+module DialogOptionMixin
+  def self.get_dialog_option(key, value, from)
+    # Return value - Support array and non-array types
+    data = value.nil? ? from[key] : value
```

Comment:
`data = value || from[key]` unless `value` can be `false`

Reply:
@bdunne these are user defined key value pairs the value could be false